### PR TITLE
[CAZ-2016] VRN search on payment matrix

### PR DESCRIPTION
**Pull request by dratka**
## Motivation and Context

Jira Card: https://eaflood.atlassian.net/browse/CAZ-2016
Users need to be able to search VRN on the payment matrix. 

## Description

Added support for full VRN search on the payment matrix. 

## How Has This Been Tested?

Tried to use the search on payment matrix (with validations check) 

## Screenshots (if appropriate):
image
image
image
image

## Checklist:

- [x] It contains only changes required by issue (does not contain other PR)
- [x] Includes link to an issue (if apply)
- [ ] I have added tests to cover my changes.

### CODE REVIEW

File: app/services/session_manipulation/clear_vrn_search.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
```
Comment (likeuwill) [inline app/services/session_manipulation/clear_vrn_search.rb]: rdoc missing

**Comment by cruddasj** [comment]
Some small updates based on the latest UI designs;

Please can we move the "clear" link below the text input and name it "Clear search" to appear like the below.
image

The text "VRN not found" will need to change (I'm currently working to get what the text should say). I'd suggest this gets merged in - then I'll raise a PR once the wording is available with the change.

**Comment by dratka** [comment]
Updated link:
image

**Comment by kingarj** [comment]
@dratka, @likeuwill Does this need to be re-reviewed?

**Pull request merged**